modules/nixos/buildbot.nix

```diff
@@ -43,11 +43,11 @@
     evalWorkerCount = 32;
     jobReportLimit = 100;
     workersFile = config.sops.secrets.buildbot-nix-workers.path;
-    cachix = {
-      enable = true;
-      name = "nix-community";
-      auth.authToken.file = config.sops.secrets.cachix-auth-token.path;
-    };
+    #cachix = {
```

Comment:
We have watch store on this machine so we can live without the buildbot cachix integration for little while.